wrappers/angular-handsontable/package.json

```diff
@@ -0,0 +1,99 @@
+{
+  "name": "@handsontable/angular",
+  "version": "6.0.1",
+  "description": "Best Data Grid for Angular with Spreadsheet Look and Feel.",
+  "author": "Handsoncode <hello@handsoncode.net> (https://handsoncode.net)",
+  "license": "MIT",
+  "homepage": "https://handsontable.com",
+  "keywords": [
+    "handsontable",
+    "component",
+    "data",
+    "table",
+    "grid",
+    "data table",
+    "data grid",
+    "spreadsheet",
+    "sheet",
+    "excel",
+    "angular",
+    "angular component",
+    "angular grid",
+    "wrapper",
+    "pro",
+    "enterprise",
+    "sort",
+    "formulas",
+    "filter",
+    "search",
+    "conditional formatting",
+    "csv"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/handsontable/handsontable.git"
+  },
+  "bugs": {
+    "url": "https://github.com/handsontable/handsontable/issues"
+  },
+  "exports": {
+    ".": "./dist/hot-table/bundles/handsontable-angular.umd.js"
```

Comment:
I noticed that. The entry can cause an error on the newest bundlers. To be checked.

Reply:
This `package.json` defines the development environment only. For the released version there is another auto-generated `package.json` file. I checked the file and I haven't found the `exports` field. 

The field is used here to instruct `import` how to consume the lib. It's used by `verify-bundles`. I'm not sure if this is the right approach to verify the version of the packages (using native import). But, I don't want to extend the review so for now it could stay as it is now.